shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -15,7 +15,8 @@
 #include "flutter/fml/memory/task_runner_checker.h"
 #include "flutter/fml/trace_event.h"
 
-const static double kRefreshRateDiffToIgnore = 4;
+// When calculating refresh rate diffrence, anything within 1 fps is ignored.
+const static double kRefreshRateDiffToIgnore = 1;
```

Comment:
Why was it 4 before, out of curiosity?

Reply:
For some reason I thought the least difference between each available FPS was 5, but I double checked the list here: https://developer.apple.com/documentation/quartzcore/optimizing_promotion_refresh_rates_for_iphone_13_pro_and_ipad_pro?language=objc and found there are refresh rates that closer. Now I just saw 15 and 16 both in the list and I think I will adjust it smaller, probably 0.1 to be safe.